python/ray/data/_internal/compute.py

```diff
@@ -61,6 +61,7 @@ def _apply(
         block_list: BlockList,
         clear_input_blocks: bool,
         name: Optional[str] = None,
+        target_block_size: Optional[int] = None,
```

Comment:
Non-blocking comment. But this function deserves a documentation for its parameters as well, to explain what's the behavior when settting and not setting `target_block_size`.

Reply:
Agreed, will update these in a follow-up PR.